### PR TITLE
[manta-pc] trigger container push on binary release

### PR DESCRIPTION
**Pull request by grenade**

### CODE REVIEW

**Comment by stechu** [comment]
@grenade  any success with the experiment?

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        with:
+          username: ${{ secrets.MANTABOT_DOCKER_USERNAME }}
+          password: ${{ secrets.MANTABOT_DOCKER_TOKEN }}
+      -
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: docker/calamari.Dockerfile
+          tags: |
+            mantabot/calamari:latest
```
Comment (Dengjianping) [inline .github/workflows/docker.yml:25]: Are we going to use `mantabot` instead of `mantanetwork`?

**Comment by Dengjianping** [comment]
@grenade You have to merge latest `manta-pc` to make it pass.

**Comment by stechu** [comment]
Closed for non-verified commit.

**Pull request closed without merging**